src/store/reducers/MovieReducer.js

```diff
@@ -7,7 +15,14 @@ const initialState = {
     page: 1,
     results: []
   },
-  currentMovie: {},
+  currentMovie: {
+    comments: {
+      total: 0,
+      total_pages: 1,
+      page: 1,
+      results: []
+    } 
+  },
```

Comment:
sobirzom da ces komentare nezavisno fetch-ovati od samog filma, onda slobodno izdvoj napolje to, samo `comments`